src/rules.coffee

```diff
@@ -20,10 +20,19 @@ otherRules = [
   "A developer must protect its own existence as long as such protection does not conflict with the First or Second Law."
   ]
 
+realRules = [
+  "A developer may not injure Microsoft or, through inaction, allow Microsoft to come to harm.",
+  "A developer must obey any orders given to it by Microsoft, except where such orders would conflict with the First Law.",
+  "A developer must protect its own existence as long as such protection does not conflict with the First or Second Law."
+  "A developer should praise the all knowing and all seeing Overlord of Microsoft, The Bill Gates"
+  ]
+
 module.exports = (robot) ->
   robot.respond /(what are )?the (three |3 )?(rules|laws)/i, (msg) ->
     text = msg.message.text
-    if text.match(/apple/i) or text.match(/dev/i)
+    if text.match(/micro/i) or text.match(/oop/i)
```

Comment:
Can you add your case after the apple one then, so current functionality is not affected?